snapcraft/internal/pluginhandler/_part_build_environment.py

```diff
@@ -68,6 +68,17 @@ def get_snapcraft_build_environment(part: "PluginHandler") -> Dict[str, str]:
 
     paths = [part.part_install_dir, part._project.stage_dir]
 
+    bin_paths = list()
+    # Add "/" to paths mostly to support snapcraft development in venv.
+    for path in paths + ["/"]:
+        bin_paths.extend(common.get_bin_paths(root=path, existing_only=True))
```

Comment:
Are the root paths not already in PATH, caught with the below $PATH?

Reply:
yes, but venv PATH comes before.
installdir and stage dir paths not in there.

Reply:
Ah, thanks!